### PR TITLE
test script to run all MCUs

### PR DESCRIPTION
**Pull request by ariel-hi**
Adding a test script to run all MCUs.

### CODE REVIEW

**Review by m-hilgendorf — APPROVED** [review]

File: tests/test-all-mcus.stanza [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
...TM32L082CZY7" "STM32L082KBU6" "STM32L082KZU6" "STM32L083CBT6" "STM32L083CZT6" "STM32L083CZU6" "STM32L083V8T6" "STM32L083VBT6" "STM32L083VZT6" "STM32L100C6U6" "STM32L100C6U6A" "STM32L151C6U6A" "STM32L151C8U6A" "STM32L151CBU6A" "STM32L151CCU6" "STM32L151CCU7" "STM32L151QDH6" "STM32L151QEH6" "STM32L151RCY6" "STM32L151RDY6" "STM32L151RDY7" "STM32L151UCY6" "STM32L151UCY7" "STM32L151V8T6A" "STM32L151VBT6A" "STM32L151VCT6" "STM32L151VDT6" "STM32L151VDY6X" "STM32L151VDY7X" "STM32L151VEY6" "STM32L151VEY7" "STM32L151ZDT6" "STM32L151ZET6" "STM32L152C6U6A" "STM32L152C8U6A" "STM32L152CBU6A" "STM32L152CCU6" "STM32L152QDH6" "STM32L152QEH6" "STM32L152RDY6" "STM32L152UCY6" "STM32L152V8T6A" "STM32L152VBT6A" "STM32L152VCT6" "STM32L152VDT6" "STM32L152VDT6X" "STM32L152VEY6" "STM32L152ZDT6" "STM32L152ZET6" "STM32L162QCH6" "STM32L162QDH6" "STM32L162RDY6" "STM32L162VCT6" "STM32L162VCT6A" "STM32L162VDT6" "STM32L162VDY6X" "STM32L162VEY6" "STM32L162ZCT6" "STM32L162ZDT6" "STM32L162ZET6" "STM32L475VCT6" "STM32L475VET6" "STM32L475VGT6" "STM32L476MEY6" "STM32L476MGY6" "STM32L476QEI6" "STM32L476QGI6" "STM32L476VCT6" "STM32L476VET6" "STM32L476VGT6" "STM32L476ZET6" "STM32L476ZGT6" "STM32L476ZGT6P" "STM32L476ZGT7P"]
```
Comment (bpellenard) [inline tests/test-all-mcus.stanza:9]: I would recommend to move them in a data file (`txt` file?), with one package per line. Then, here, read the file.

File: tests/test-all-mcus.stanza [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+val start-index = 0
+val end-index = 1
```
Comment (bpellenard) [inline tests/test-all-mcus.stanza:12]: are you sure about `end-index` value?

**Review by bpellenard — APPROVED** [review]

**Comment by ariel-hi** [comment]
A similar test script will be written for end to end testing.

File: tests/test-all-mcus.stanza (hunk 2 above)
Comment (RajeshJITX) [reply tests/test-all-mcus.stanza:12]: replace it with `length(mpns)` ?

**Pull request closed without merging**